linenoise/idf_component.yml

```diff
@@ -0,0 +1,9 @@
+version: "1.0.0"
```

Comment:
Since the original upstream version of linenoise was tagged as 1.0, perhaps we can define this as 2.0.0?